src/Init/Grind/CommRing/Basic.lean

```diff
@@ -43,6 +43,12 @@ class CommRing (α : Type u) extends Add α, Mul α, Neg α, Sub α, HPow α Nat
   pow_succ : ∀ a : α, ∀ n : Nat, a ^ (n + 1) = (a ^ n) * a
   ofNat_succ : ∀ a : Nat, OfNat.ofNat (α := α) (a + 1) = OfNat.ofNat a + 1 := by intros; rfl
 
+-- We reduce the priority of these parent instances,
+-- so that in downstream libraries with their own `CommRing` class,
+-- the path `CommRing -> Add` is found before `CommRing -> Lean.Grind.CommRing -> Add`.
```

Comment:
Is there/will there be a `CommRing -> Lean.Grind.CommRing` path?

Reply:
Yes, it's already there in nightly-testing.

Reply:
The path that this PR avoids is still defeq, but that still managed to trip up the `move_add` tactic.